scripts/gen_binding.py

```diff
@@ -523,12 +523,15 @@ def normalize_arg(arg):
     const [t0, b0] = recorded_stat
     const dt = performance.now() - t0
     const db = fl.bytesUsed.native() - b0
+    const ops = BigInt([{','.join(js_tensor_args)}].reduce((ops, t) => ops + t.elements, 0))
```

Comment:
Not all ops are created equal so there is probably a more intelligent way to track "useful" ops, but hopefully this is a sufficient approximation for now

Reply:
is this an approximation of `flops`?  if so, I think we should jump into immediately making a registry to calculate it exactly for each operation.  most are going to be point-wise (as you have here), but the ones that really count will not be (matmul, conv)

Reply:
I'm probably not the best person to fill out that registry as I'm not certain the exact complexity of every method, but I can give it a shot.